gkr_iop/src/precompiles/lookup_keccakf.rs

```diff
@@ -558,6 +558,20 @@ where
 {
     type Trace = KeccakTrace;
 
+    fn fixed_witness_group(&self) -> RowMajorMatrix<E::BaseField> {
```

Comment:
👍  

Not this line, but per our offline discussion, to complete functionality, we need expand (i.e. `iter().cycle()` in rust) fixed poly dynamically via num_instances. I will come back to this part when I make the e2e integration